source/features/select-notifications.tsx

```diff
@@ -14,18 +14,22 @@ import {
 	GitPullRequestDraftIcon,
 	GitPullRequestIcon,
 	IssueOpenedIcon,
+	RocketIcon,
 	XCircleIcon,
+	XIcon,
 } from '@primer/octicons-react';
 
 import features from '.';
 
 const filters = {
 	'Pull requests': ':is(.octicon-git-pull-request, .octicon-git-pull-request-closed, .octicon-git-pull-request-draft, .octicon-git-merge)',
 	Issues: ':is(.octicon-issue-opened, .octicon-issue-closed)',
-	Open: ':is(.octicon-issue-opened, .octicon-git-pull-request)',
+	Deployments: ':is(.octicon-rocket, .octicon-x)',
```

Comment:
`.octicon-x` will match every notification for failed CI jobs, not just deployments, right?

While I like the "Run failed" filter, this one is probably too broad. Is there an easy way to limit it to "failed deployments" only?

Reply:
It seems like GH doesn't really distinguishes `Deployments` from any other `CI activity` triggered by a workflow in general as `.octicon-x` does indeed match every notification for failed CI. I am just trying to figure out if it will match any other type, but for any of my internal tests I could not find it.

I am going to propose, instead of trying to handle specifics of failed runs for `Deployments`, renaming it to `CI activity`. It is a somewhat broad filter, but do you think it loses its value?